### PR TITLE
Opr 163 bugfix

### PR DESCRIPTION
**Pull request by LiekeSchoenmaker**
SCOPE: 
- Solve bug for loading nullable values columns studyAcronym, studyGender, treatmentApproachesDrugClass & treatmentApproachesTherapy
- Solve bug for correct interpretation of treatmentApproachesDrugClass & treatmentApproachesTherapy

### CODE REVIEW

**Review by cbruel — APPROVED** [review]
I'm not a software developer, but changes look good to me!

File: ckb-importer/src/main/java/com/hartwig/serve/ckb/datamodel/treatmentapproaches/RelevantTreatmentApproachesFactory.java [hunk 1]
```diff
@@ -26,8 +27,9 @@ public static List<DrugClassTreatmentApproach> extractDrugTreatmentApproaches(@N
         for (TreatmentApproachInfo treatmentApproachInfo : treatmentApproachInfos) {
             DrugClassTreatmentApproach resolvedRelevantTreatmentApproaches =
                     resolveDrugTreatmentApproaches(ckbJsonDatabase, treatmentApproachInfo);
-            relevantTreatmentApproach.add(resolvedRelevantTreatmentApproaches);
-        }
+            if (resolvedRelevantTreatmentApproaches != null) {
+                relevantTreatmentApproach.add(resolvedRelevantTreatmentApproaches);
+            }        }
```
Comment (mdenburger) [inline ckb-importer/src/main/java/com/hartwig/serve/ckb/datamodel/treatmentapproaches/RelevantTreatmentApproachesFactory.java:32]: Would be good to fix the funky formatting :)

File: algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java [hunk 2]
```diff
@@ -66,10 +66,13 @@ public Set<ActionableEntry> create(@NotNull CkbEntry entry, @NotNull String sour
                                 sourceGene,
                                 sourceEvent)) {
 
+                            Set<String> sourceUrls = Sets.newHashSet();
+                            sourceUrls.add("https://ckbhome.jax.org/profileResponse/advancedEvidenceFind?molecularProfileId=" + entry.profileId());
```
Comment (mdenburger) [inline algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java:70]: You could use `Collections.singleton("<url>")` instead

**Review by mdenburger — APPROVED** [review]
Approved with two minor notes.

File: ckb-importer/src/main/java/com/hartwig/serve/ckb/datamodel/treatmentapproaches/RelevantTreatmentApproachesFactory.java (hunk 1 above)
Comment (LiekeSchoenmaker) [reply ckb-importer/src/main/java/com/hartwig/serve/ckb/datamodel/treatmentapproaches/RelevantTreatmentApproachesFactory.java:32]: solved

File: algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java (hunk 2 above)
Comment (LiekeSchoenmaker) [reply algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java]: solved

**Pull request merged**